src/components/claim/ClaimModal.tsx

```diff
@@ -24,6 +24,7 @@ import { ActionButton } from './ActionButton'
 import { useIsOldSwaprLp } from '../../hooks/swpr/useIsOldSwaprLp'
 import { ConvertFlow } from './ConvertFlow'
 import useDebounce from '../../hooks/useDebounce'
+import { AddTokenButton } from '../AddTokenButton/AddTokenButton'
```

Comment:
One small thing I would add is that naming AddTokenButton.tsx to index.tsx since it will be just cleaner import without second AddTokenButton